src/score.js

```diff
@@ -186,13 +187,9 @@ export const makeCandidateScoreCalc = (config, hostname, chatDomain, apiSlug, ap
 
         let responseText = "";
 
-        // @ts-expect-error FIXME
-        const isEligible = makeIsEligible(repNominate);
-
-        if (config.debug) {
+        if (config.verbose) {
```

Comment:
Everything here is a huge log dump - moved to `verbose` level. We should probably log minimal info in prod, though